defaults/main.yml

```diff
@@ -19,7 +19,7 @@ idr_jupyter_hub_extra_docker_args: ""
 idr_jupyter_hub_image: "imagedata/jupyterhub-githubauth:latest"
 # Docker Swarm requires a modified version of DockerSpawner:
 #idr_jupyter_hub_image: "manics/jupyterhub-githubauth:latest"
-idr_jupyter_notebook_image: "imagedata/jupyter-docker:latest"
+idr_jupyter_notebook_image: "manics/jupyter-docker:merge"
```

Comment:
Yes, but https://github.com/IDR/jupyter-docker/pull/7#issuecomment-298029010 is required for testing interactively with vagrant (testing with docker uses a non-functional dummy image to save disk space on travis).